docs/_docs/configuration.md

```diff
@@ -601,12 +613,13 @@ collections:
     output:   true
 
 # Handling Reading
-safe:         false
-include:      [".htaccess"]
-exclude:      ["node_modules", "vendor/bundle/", "vendor/cache/", "vendor/gems/", "vendor/ruby/"]
-keep_files:   [".git", ".svn"]
-encoding:     "utf-8"
-markdown_ext: "markdown,mkdown,mkdn,mkd,md"
+safe:                false
+include:             [".htaccess"]
+exclude:             ["node_modules", "vendor/bundle/", "vendor/cache/", "vendor/gems/", "vendor/ruby/"]
```

Comment:
Note: To be fixed on `master` or via another pull

`exclude:` array is now `["Gemfile", "Gemfile.lock", "node_modules", "vendor/bundle/", "vendor/cache/", "vendor/gems/", "vendor/ruby/"]`

/cc @DirtyF